### PR TITLE
Testing/url negtive testing cy scripts

### PR DESCRIPTION
**Pull request by kprisAutomation**
url negtive testing cy test cases and test data

### CODE REVIEW

File: cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+
+  it('Error messages display according to the language selected', function () {
+    //switch to Spanish
+    cy.get('[aria-label="language"]').click()
+    cy.get('[id^=cdk-overlay]').within(($menuLanguage) => {
+      cy.get('button[role="menuitem"]')
+        .contains(testingData.errorMessages.displayLanguageSpanish)
+        .click()
+    })
+    cy.wait(3000) //wait for language to refresh
```
Comment (leomendoza123) [inline cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js:276]: We can avoid using the wait 
```suggestion
   cy.get('[lang="es"]') //wait for language to refresh
```

File: cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+    cy.get('#cancel-websites-button').click()
+
+    //switch back to English
+    cy.get('[aria-label="idioma"]').click()
+    cy.get('[id^=cdk-overlay]').within(($menuLanguage1) => {
+      cy.get('button[role="menuitem"]')
+        .contains(testingData.errorMessages.displayLanguageEnglish)
+        .click()
+    })
+    cy.wait(3000) //wait for language to refresh before Log out
```
Comment (leomendoza123) [inline cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js:303]: We can avoid using the wait 
```suggestion
   cy.get('[lang="en"]') //wait for language to refresh
```

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

File: cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+      .type(testingData.sidebarWebsitesURL.titleURL)
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.duplicateURL)
+    cy.get('#save-websites-button').click()
+
+    //add same URL via API
+    // const jsonFilePath=userData.cyUserMemmerAPI.curlPostWebsitePath
+    const endpoint =
+      'https://api.qa.orcid.org/v3.0/' +
```
[the remark this answers is not in the document]
Comment (kprisAutomation) [reply cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js:462]: Done.

File: cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+
+    //verify the URL becomes the link title displayed in my-orcid
+    cy.get('#websites-panel')
+      .within(($section) => {
+        cy.get('[class="line"]')
+      })
+      .should('contain', testingData.sidebarWebsitesURL.duplicateURL)
+
+    //clean up state
+    cy.cleanWebsites()
```
[the remark this answers is not in the document]
Comment (kprisAutomation) [reply cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js:36]: Done.

File: cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js [hunk 5]
```diff
@@ -0,0 +1,506 @@
...
+    //verify the URL field is required
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.invalidURL
+    )
+
+    cy.get('#cancel-websites-button').click()
+  })
+
+  it('Arabic characters in title field are allowed', function () {
```
[the remark this answers is not in the document]
Comment (kprisAutomation) [reply cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js:354]: Done, moved these valid scenarios to the happy path script.

File: cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js (hunk 1 above)
Comment (kprisAutomation) [reply cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js:276]: Done

File: cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js (hunk 2 above)
Comment (kprisAutomation) [reply cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js:303]: Done

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**